dnf-behave-tests/features/vars-releasever.feature

```diff
@@ -86,3 +86,63 @@ Scenario: Releasever is substituted in baseurl via a value detected from 'system
     And Transaction is following
         | Action        | Package                       |
         | install       | setup-0:2.12.1-1.fc29.noarch  |
+
+@destructive
+Scenario: Releasever is substituted in baseurl using vars loaded from location (host or installroot) like used repos
+  Given I do not set releasever
+    And I do not set reposdir
+    And I copy directory "{context.dnf.repos_location}/dnf-ci-fedora" to "/temp-repos/base-f0123"
+    And I create and substitute file "/etc/yum.repos.d/test.repo" with
```

Comment:
you can create repo file right in the final destination by using leading double slash (that means the path will not be prepended by installroot):
```
And I create and substitute file "//etc/yum.repos.d/test.repo" with
```
Than the moving part on lines 111-112 is obsolete.